docker-compose.yaml

```diff
@@ -210,10 +228,16 @@ services:
       - POSTGRES_DB=hive
       - POSTGRES_URL=postgres:5432
       - JUPYTERHUB_ADMIN_PASSWORD=testpassword123
+      - NETWORK_NAME=cdm-jupyterhub-network
+      - JUPYTERHUB_USER_IMAGE=cdm-jupyterhub-cdm_jupyterhub:latest
```

Comment:
Is there eventually going to be a small spawner image and a user image or are you thinking of just having the 1

Reply:
spawner image will be this one since it has all the packages users need. Maybe having a light image for the jupterhub server.

Reply:
I think we mean different things by "spawner". When I say spawner I'm talking about whatever is responsible for spawning the user containers

Reply:
yea 
> Maybe having a light image for the jupterhub server..